ccd-definition/CaseEventToComplexTypes/CreateClaimCUI.json

```diff
@@ -7,12 +7,28 @@
     "FieldDisplayOrder": 11,
     "DisplayContext": "OPTIONAL"
   },
+  {
```

Comment:
I think these changes need to be in their own HNL-nonprod.json file else I think these will go straight into production when merged.